polyfill/spatial-navigation-polyfill.js

```diff
@@ -1059,62 +1083,111 @@
    * @returns {Points} The exit point from the search origin and the entry point from a candidate
    */
   function getEntryAndExitPoints(dir = 'down', rect1, rect2) {
-    const points = {entryPoint:[0,0], exitPoint:[0,0]};
+    const points = {entryPoint: {x: 0, y: 0}, exitPoint:{x: 0, y: 0}};
 
-    // Set direction
-    switch (dir) {
-    case 'left':
-      points.exitPoint[0] = rect1.left;
-      points.entryPoint[0] = (rect2.right < rect1.left) ? rect2.right : rect1.left;
-      break;
-    case 'up':
-      points.exitPoint[1] = rect1.top;
-      points.entryPoint[1] = (rect2.bottom < rect1.top) ? rect2.bottom : rect1.top;
-      break;
-    case 'right':
-      points.exitPoint[0] = rect1.right;
-      points.entryPoint[0] = (rect2.left > rect1.right) ? rect2.left : rect1.right;
-      break;
-    case 'down':
-      points.exitPoint[1] = rect1.bottom;
-      points.entryPoint[1] = (rect2.top > rect1.bottom) ? rect2.top : rect1.bottom;
-      break;
-    }
+    if (startingPoint) {
```

Comment:
When there is `startingPoint`,   It pass `statingPoint` as parameter `getEntryAndExitPoints(dir, **point**, element);`
But `getEntryAndExitPoints` function never use the `rect1` parameter. 
Instead, refer to `startingPoint` again. It looks strange.

Reply:
I'll fix it. Thanks.